src/main/webapp/user-private-comments.jsp

```diff
@@ -20,19 +20,19 @@
   </head>
   <script>
     var privateCommentsDataJSON = <%=new Gson().toJson(privateCommentsMap)%>;
-    function init(privateCommentID) {
+    function init(privateCommentTaskID) {
         console.log("here!");
         const editor = new EditorJS({
-        holderId : "box-" + privateCommentID,
+        holderId : "box-" + privateCommentTaskID,
         autofocus: true,
-        data: privateCommentsDataJson[privateCommentID],
+        data: privateCommentsDataJson[privateCommentTaskID],
         tools: {
           header: Header,
         },
       });
-      console.log("here 2 !");
+      return editor;
     }
-    function save() {
+    function save(editor) {
```

Comment:
Pass editor so it knows which editor is being saved as there is one for every task.